pkg/anago/stage.go

```diff
@@ -500,6 +626,33 @@ func (d *DefaultStage) GenerateChangelog() error {
 	})
 }
 
+func (d *DefaultStage) GenerateBillOfMaterials() error {
+	for _, version := range d.state.versions.Ordered() {
+		artifactsList, err := d.impl.ListArtifacts(version)
+		if err != nil {
+			return errors.Wrap(err, "getting artifacts list")
+		}
+		logrus.Infof("Artifacts list: %+v", artifactsList)
```

Comment:
I'm not sure if we need this. Maybe we can list them directly during iteration to be better readable.

Reply:
(yes it's just a tmp debugger I'm using to locate the artifacts : )

Reply:
Done, removed